doc/source/serve/index.md

```diff
@@ -121,9 +126,89 @@ Because it's built on top of Ray, you can run it anywhere Ray can: on your lapto
 
 :::
 
+
+## How can Serve help me as a...
+
+:::{dropdown} Data scientist
+:animate: fade-in-slide-down
+
+Serve is easy to use. You can test your models (and your entire deployment graph) on your local machine before deploying it to production on a cluster. You don't need to know heavyweight Kubernetes concepts or cloud providers to use Serve.
+
+:::
+
+:::{dropdown} ML engineer
+:animate: fade-in-slide-down
+
+Serve helps you scale out the deployment and run them reliably in an efficient, cost-saving manner. Serve offers a first-class model composition API to help you combine models together to build end-to-end application. Additionally, Serve natively run on Kubernetes which minimal operation overhead.
+
+:::
+
+:::{dropdown} ML Platform engineer
+:animate: fade-in-slide-down
+
+Serve specializes in ML model serving and can be a reliably component in your ML Platform stack.
+Serve supports arbitrary Python code and therefore integrates well with the MLOps ecosystem. You can use it with model optimizers (ONNX, TVM), model monitoring systems (Seldon Alibi, Arize), model registries (MLFlow, Weights and Biases), machine learning frameworks (XGBoost, Scikit-learn), data app UIs (Gradio, Streamlit), and Web API frameworks (FastAPI, gRPC).
+
+:::
+
+
+## How does Serve compare to X
+
+:::{dropdown} TFServing, TorchServe, ONNXRuntime
+:animate: fade-in-slide-down
+
+Ray Serve is *framework agnostic*, you can use any Python framework and libraries.
+We believe data scientists are not bounded a particular machine learning framework.
+They use the best tool available for the job.
+
+Compared to these framework specific solution, Ray Serve doesn't perform any optimizations
+to make your ML model run faster. However, you can still optimize the models yourself
```

Comment:
Compared to these framework-specific solutions, Ray Serve doesn't perform any optimizations
to make your ML model run faster.